Cargo.toml

```diff
@@ -93,6 +93,7 @@ base64 = "0.10"
 safemem = "0.3"
 cop_datatype = { path = "components/cop_datatype" }
 smallvec = { version = "0.6", features = ["union"] }
+procinfo = { git = "https://github.com/danburkert/procinfo-rs", rev = "3e5c94a9ef7ff237a2602770aa1b687e6520e896" }
```

Comment:
This is the newest revision, with a PR from @overvenus merged.
We need the APIs provided by this PR.